aws-amplify/amplify-js
# feat(@aws-amplify/ui-components): amplify-s3-image component

_Description of changes:_
- `amplify-s3-image` component
 
Validation steps:
- From `amplify-js`, `cd packages/amplify-ui-components`
- Copy or update `aws-exports.js` file with new storage config, ping me if need be
- Run `yarn` then `yarn start`
- Storage example should show up first, if not navigate to Storage sample

Usage:
`<amplify-s3-image imgKey="amplify-logo.png">`
- `imgKey` represents the key of the image inside the Amplify CLI configured S3 bucket

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
